app/MMB/src/scenes/tickets/boardingPasses/BoardingPassMultiCity.js

```diff
@@ -0,0 +1,56 @@
+// @flow strict
+
+import * as React from 'react';
+import { Translation } from '@kiwicom/mobile-localization';
+import { createFragmentContainer, graphql } from '@kiwicom/mobile-relay';
+import { TextIcon, StyleSheet, Color } from '@kiwicom/mobile-shared';
+import idx from 'idx';
+
+import type { BoardingPassMultiCity as BookingType } from './__generated__/BoardingPassOneWay.graphql';
```

Comment:
Is it the right file?

Reply:
Ohh, the dangers of copy/paste I should disable it on my machine 😖

Reply:
That's what code reviews are for :) two sets of eyes is better than one ;-)